k9mail/src/main/java/security/planck/ui/about/AboutActivity.kt

```diff
@@ -117,62 +127,99 @@ class AboutActivity : K9Activity() {
-            "Bouncy Castle" to "https://github.com/Kotlin/kotlinx.serialization",
-            "Androidx libraries" to "https://github.com/androidx/androidx",
-
-            "AppAuth" to "https://github.com/openid/AppAuth-Android.git",
-            "JWTDecode" to "https://github.com/auth0/JWTDecode.Android.git",
-            "Okio" to "https://github.com/square/okio.git",
-            "Commons IO" to "https://github.com/apache/commons-io.git",
-            "Ant" to "https://github.com/apache/ant",
-            "James" to "https://github.com/apache/james-mime4j/tree/master",
-            "Java Concurrency In Practice" to "https://jcip.net/",
-            "Moshi" to "https://github.com/square/moshi.git",
-            "TokenAutocomplete" to "https://github.com/splitwise/TokenAutoComplete.git",
-            "ShowCaseView" to "https://github.com/amlcurran/ShowcaseView.git",
-            "Timber" to "https://github.com/JakeWharton/timber.git",
-            "Butterknife" to "https://github.com/JakeWharton/butterknife.git",
-            "Renderers" to "https://github.com/pedrovgs/Renderers.git",
-            "Glide" to "https://github.com/bumptech/glide.git",
-            "AppIntro" to "https://github.com/AppIntro/AppIntro.git",
-            "SwipeLayout" to "https://github.com/daimajia/AndroidSwipeLayout.git",
-            "Dexter" to "https://github.com/Karumi/Dexter.git",
-            "SafeContentResolver" to "https://github.com/cketti/SafeContentResolver.git",
-            "CircleImageView" to "https://github.com/hdodenhof/CircleImageView.git",
-            "Jsoup" to "https://github.com/jhy/jsoup.git",
-            "Preferencex" to "https://github.com/takisoft/preferencex-android.git",
-            "Groupie" to "https://github.com/lisawray/groupie.git",
-            "Biweekly" to "https://github.com/mangstadt/biweekly.git",
-
-            "MiniDNS" to "https://github.com/MiniDNS/minidns.git",
-            "jdom" to "https://github.com/hunterhacker/jdom.git",
-            "Dagger Hilt" to "https://github.com/google/dagger/tree/master/java/dagger/hilt",
-            "Bouncy Castle" to "https://www.bouncycastle.org/java.html",
-
-            "Truth" to "https://github.com/google/truth.git",
-            "Spoon" to "https://github.com/square/spoon",
-            "Mockito" to "https://github.com/mockito/mockito.git",
-            "Mockito Kotlin" to "https://github.com/mockito/mockito-kotlin.git",
-            "Mockk" to "https://github.com/mockk/mockk.git",
-            "Cucumber Android" to "https://github.com/cucumber/cucumber-android.git",
-            "Cucumber Picocontainer" to "https://github.com/cucumber/cucumber-jvm/tree/main/cucumber-picocontainer",
-            "Robolectric" to "https://robolectric.org/",
-            "JUnit" to "https://junit.org/junit4/",
-            "Barista" to "https://github.com/AdevintaSpain/Barista.git",
+        private val USED_LIBRARIES = listOf(
```

Comment:
I think for now this is fine, for the new applications it would be great if we could auto build this list automatically from dependencies instead of doing it manually, as manual is error prone. For Android on corev3 probably too complex to make this list compile automatically.

Reply:
If needed I could try some experiments, but it might take some time.